deploy/97_fixtures.ts

```diff
@@ -14,15 +16,11 @@ const func: DeployFunction = async function (env: HardhatRuntimeEnvironment) {
   console.table([alice, bob, treasury]);
 
   const lusd = await get('LUSD');
-  const vaults = {
-    yearn: await get('Yearn_LUSD_Vault'),
-    liquity: await get('Liquity_Amethyst_Vault'),
-  };
 
-  for (const vaultKey in vaults) {
+  for (const prefix of VAULT_PREFIXES) {
```

Comment:
why not just use the vault's full name?

Reply:
because in the other files i use need to grab an instance of the strategy contracts etc of each vault

no point declaring two separate arrays

Reply:
oh, I see!! I don't think this is cool. You're importing a variable from one deploy script to the other. That seems like a very weird coupling

Reply:
i can put it in a separate file in the deploy folder any suggestions for a name?

Reply:
I don't see anything wrong with duplication, but we can always go with the traditional helpers file

Reply:
we have a folder called `helpers`,

I dont think `helpers/helper.ts` is that nice in this case, `helpers/index.ts` also not right because of the way the deploy files are structured. How about `helpers/constants.ts`?

Reply:
nvm I don't think changing this is worth it

```
Error: ERROR processing ~/solidity_contracts/deploy/helpers/constants.ts:
TypeError: deployScript.func is not a function
```

The way hardhat executes all the files in that folder we would have to add a default function and then tell hardhat to skip it in deployment

Super ugly & better off as it is at the moment imo

If you check the tags of where it's used, the other two fixtures depend on this initial fixture to run so they are coupled anyway